hack/release/common.sh

```diff
@@ -29,6 +31,20 @@ versionData(){
   RELEASE_MINOR_VERSION="v${RELEASE_VERSION_MAJOR}.${RELEASE_VERSION_MINOR}"
 }
 
+snapshot() {
+  RELEASE_VERSION=$1
+  echo "Release Type: $(releaseType "${RELEASE_VERSION}")
```

Comment:
Do we need this releaseType function now that we are differentiating between snapshot and official releases with different functions?

Reply:
I'll check if there's anywhere else its used but probably not. With some more refactoring we can definitely drop it.